g4f/Provider/bing/create_images.py

```diff
@@ -161,31 +161,15 @@ def read_images(html_content: str) -> List[str]:
         raise RuntimeError("No images found")
     return images
 
-async def create_images_markdown(cookies: Dict[str, str], prompt: str, proxy: str = None) -> str:
-    """
-    Creates markdown formatted string with images based on the prompt.
-
-    Args:
-        cookies (Dict[str, str]): Cookies to be used for the session.
-        prompt (str): Prompt to generate images.
-        proxy (str, optional): Proxy configuration.
-
-    Returns:
-        str: Markdown formatted string with images.
-    """
-    async with create_session(cookies) as session:
-        images = await create_images(session, prompt, proxy)
-        return format_images_markdown(images, prompt)
-
-def get_cookies_from_browser(proxy: str = None) -> Dict[str, str]:
+def get_cookies_from_browser(proxy: str = None) -> dict[str, str]:
```

Comment:
Consider adding type hints to the `proxy` parameter of the `get_cookies_from_browser` function.